src/main/java/cinema/model/Order.java

```diff
@@ -0,0 +1,67 @@
+package cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany
+    private List<Ticket> tickets;
+    @Column(name = "order_date")
+    private LocalDateTime orderDate;
+    @OneToOne
+    private User user;
```

Comment:
user may have more than one Order

Reply:
Fixed. But I have a question. We formed an order from the shopping cart. After that, we add order to the database and after we cleared our shopping cart. And if we choose another tickets, we will form another order. So, by one session we will form one order for one user. So why `ManyToOne` assignment prevailed over `OneToOne` in this case?

Reply:
It's doesn't relate to whether we are doing this during one session or not(if you are talking about hibernate session). This annotation  discribe the relations between entities. One user can have multiple orders;